pm.pl

```diff
@@ -43,8 +43,7 @@
 
 url_found(URL) :- dependency(URL, _).
 probably_add_dep(Handle, URL) :- 
-    format(atom(Formatted), 'dependency("~w", "latest").', [URL]),
-    writeln(Handle, Formatted).
+    write_term(Handle, dependency(URL, "latest"), [quoted(true), fullstop(true), nl(true)]).
```

Comment:
May want to refactor this into a separate predicate at some point to factor out all those common options in the writeterm call